ui/components/app/modals/confirm-remove-account/confirm-remove-account.stories.js

```diff
@@ -22,7 +22,14 @@ export default {
         },
       },
       options: {},
-      methods: [...Object.values(EthMethod)],
+      methods: [
```

Comment:
Any reason why were are specifying them explicitly now?

Reply:
EthMethods has UserOp methods as well. We should probably update the keyring api to separate the methods.